lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -220,7 +220,7 @@ defmodule ExUnit.Runner do
     EM.test_started(config.manager, test)
 
     if is_nil(test.state) do
-      capture_log? = Map.get(tags, :capture_log, false)
+      capture_log? = Map.get(tags, :capture_log, Application.fetch_env!(:ex_unit, :capture_log))
```

Comment:
Had no idea that existed, I need to check out the 1.1 changelog :). But I'm expecting very few test cases to be annotated with `:capture_log`, so the env will have to be fetched almost always anyhow, not sure if it's worth it. But no strong opinion either way.